cpp/Platform.Setters/Setter[TResult, TDecision].h

```diff
@@ -1,46 +1,47 @@
-﻿namespace Platform::Setters
-{
-    template <typename ...> class Setter;
-    template <typename TResult, typename TDecision> class Setter<TResult, TDecision> : public SetterBase<TResult>
-    {
-        private: TDecision _trueValue = 0;
-        private: TDecision _falseValue = 0;
-
-        public: Setter(TDecision trueValue, TDecision falseValue, TResult defaultValue)
-            : SetterBase(defaultValue)
-        {
-            _trueValue = trueValue;
-            _falseValue = falseValue;
-        }
-
-        public: Setter(TDecision trueValue, TDecision falseValue) : this(trueValue, falseValue, 0) { }
-
-        public: Setter(TResult defaultValue) : SetterBase(defaultValue) { }
-
-        public: Setter() { }
-
-        public: TDecision SetAndReturnTrue(TResult value)
-        {
-            _result = value;
-            return _trueValue;
-        }
-
-        public: TDecision SetAndReturnFalse(TResult value)
-        {
-            _result = value;
-            return _falseValue;
-        }
-
-        public: TDecision SetFirstAndReturnTrue(IList<TResult> &list)
-        {
-            _result = list[0];
-            return _trueValue;
-        }
-
-        public: TDecision SetFirstAndReturnFalse(IList<TResult> &list)
-        {
-            _result = list[0];
-            return _falseValue;
-        }
-    };
-}
+﻿namespace Platform::Setters
+{
+    template <typename ...> class Setter;
+    template <typename TResult, typename TDecision> class Setter<TResult, TDecision> : public SetterBase<TResult>
+    {
+        private: TDecision _trueValue = 0;
+        private: TDecision _falseValue = 0;
+
+        using base = SetterBase<TResult>;
+
+        public: Setter(TDecision trueValue, TDecision falseValue, TResult defaultValue)

```

Comment:
can be use
```cpp
Setter(TDecision trueValue, TDecision falseValue, TResult defaultValue) : 
    /* some spaces */   trueValue(trueValue), // also can use for TDecision
                                   falseValue(falseValue), 
                                   defaultValue(std::move(defaultValue))
```

Reply:
We can write in single line.

Reply:
long

Reply:
Yes, but it is ok, where are big screens and line wrap read mode.